docs/zarr.md

```diff
@@ -0,0 +1,129 @@
+            │
+            └── 0             # Multiscale, labeled image. The name is unimportant but is registered in the "labels" group above.
+                ├── .zgroup   # Zarr Group which is both a multiscaled image as well as a labeled image.
+                ├── .zattrs   # Metadata of the related image and as well as display information under the "image-label" key.
+                │
+                ├── 0         # Each multiscale level is stored as a separate Zarr array, as above, but only integer values
+                │   ...       # are supported.
+                └── n
+```
+
+See [OME-Zarr 0.4 spec](https://ngff.openmicroscopy.org/latest/index.html#image-layout) for details.
+
+## Conversion to Zarr
+
+You can easily convert image stacks manually with the [WEBKNOSSOS CLI](https://docs.webknossos.org/cli).
+The CLI tool expects all image files in a single folder with numbered file names.
+After installing, you can convert image stacks to Zarr datasets with the following command:
+
+```shell
+pip install webknossos
+
+webknossos convert \
+  --voxel-size 11.24,11.24,25 \
+  --name my_dataset \
+  --data-format zarr3 \
+  data/source data/target
+```
+
+This snippet converts an image stack that is located in directory called `data/source` into a Zarr dataset which will be located at `data/target`.
+It will create a so called `color` layer containing your raw greyscale/color image.
+The supplied `--voxel-size` is specified in nanometers.
+
+Read the full documentation at [WEBKNOSSOS CLI](https://docs.webknossos.org/cli).
+
+### Conversion with Python
+
+You can use the free [WEBKNOSSSO Python library](https://docs.webknossos.org/webknossos-py) to convert image stacks to Zarr or integrate the conversion as part of an existing workflow. 
+
+```python
+from webknossos import Dataset
+from webknossos.dataset import COLOR_CATEGORY
+
+def main() -> None:
+    """Convert a folder of image files to a WEBKNOSSOS dataset."""
+    dataset = Dataset.from_images(
+        input_path=INPUT_DIR,
+        output_path=OUTPUT_DIR,
+        voxel_size=(11, 11, 11),
+        layer_category=COLOR_CATEGORY,
+        compress=True,
+        data_format = Dataformat.Zarr
+    )
+
+    print(f"Saved {dataset.name} at {dataset.path}.")
+
+    # dataset.upload()
+
+
+if __name__ == "__main__":
+    main()
+```
+
+Read the full example in the WEBKNOSSOS [Python library documentation](https://docs.webknossos.org/webknossos-py/examples/create_dataset_from_images.html).
+
+## Time-Series and N-Dimensional Datsets
+
+WEBKNOSSOS also supports loading n-dimensional datasets, e.g. 4D = time series of 3D microscopy.
+This feature in currently only supported for Zarr dataset due to their flexbile structure and design for n-dimensional data.
+
+## Performance Considerations
+To get the best streaming performance for Zarr datasets consider the following settings.
+
+- Use chunk sizes of 32 - 128 voxels^3
+- Enable sharding
```

Comment:
```suggestion
```
Not in Zarr 2 yet.

Reply:
Add a note about the upcoming Zarr3 support and that it will be default for WK in the future.

Reply:
Done